### ISSUE TITLE
[Shadowrun5th] add the data-i18n tags

### ISSUE THREAD

**Issue by dyvim** [Roll20/roll20-character-sheets#4228]
Hello Jens, 

Can you please add the data-i18n tags so that your superb character sheet can be translated in other languages?

Shadowrun is really appreciated in other countries and languages and no other shadowrun sheet is available in other languages.

It will be very appreciated. 

Thanks by advance.

**Comment by Anduh** [comment]
@dyvim you could yourself get this started, there exist instructions and examples on how to do this, no programming knowledge is needed,  just follow some simple rules in inserting the tags. https://wiki.roll20.net/Character_Sheet_i18n

**Comment by dyvim** [comment]
Yes I know that and I know the rules but I don't want to start this without the creator being noticed

**Comment by clevett** [comment]
Fyi,

I put in the pull request for the Shadowrun 5e sheet I've been working on. It has the translation keys to be multilingual. Downside is its a whole new sheet so you may not want to start over. The translation keys might help you get started on adding ones to the Advanced sheet.

- Cassie

**Comment by dyvim** [comment]
I'll check that (and probably translate your new sheet ASAP). Thx

**Comment by clevett** [comment]
That would be very helpful! I could then adjust the CSS to be more accommodating for multilingual.

**Comment by clevett** [comment]
@dyvim 

The new [Shadowrun 5th Edition sheet](https://github.com/Roll20/roll20-character-sheets/tree/master/Shadowrun5thEdition) is now live and includes the translations files.

**Comment by clevett** [comment]
@dyvim 

Are we good to close this request? Do you still want to translate the Advanced sheet? It appears to have not been updated in 6 months.

Thanks,
Cassie

**Comment by dyvim** [comment]
Yes you can close the request. It seems I will have to do it myself if I want it to be done.
